### PR TITLE
Postgres save

### PR DESCRIPTION
**Pull request by AmiraVomir**
Added postgres connection
changed cassandra connection

### CODE REVIEW

File: scripts/dataGen.js [hunk 1]
```diff
@@ -40,13 +40,13 @@ const dataGen = (writer, encoding, callback) => {
...
+//   writeData.end();
+//   const ending = new Date().getTime() - start.getTime();
+//   console.log(`Seeding Completed! It took: ${Math.floor(ending / 60000)}mins and ${((ending % 60000) / 1000).toFixed(0)}secs`);
+// });
 
-exports.default = {
+module.exports = {
   dataGen,
   writeData,
   start,
```
Comment (jgenso1690) [inline scripts/dataGen.js:52]: add an empty line

File: db/postgreSQL.js [hunk 2]
```diff
@@ -30,12 +33,17 @@ const Product = sequelize.define('mydb', {
...
+  })
+  .then(() => pgProduct.query("COPY products FROM '/home/hieuho/Hack Reactor/sdc/suggested-module/data.csv' DELIMITER ',' CSV HEADER"))
+  .then(() => {
+    const ending = new Date().getTime() - seed.start.getTime();
+    return console.log(`Seeding Completed! It took: ${Math.floor(ending / 60000)}mins and ${((ending % 60000) / 1000).toFixed(0)}secs`);
+  })
+  .catch((err) => console.log('Connection or Seeding Error', err));
 
-// COPY products_by_shop FROM '/home/hieuho/Hack Reactor/sdc/suggested-module/data.csv'DELIMITER ',' CSV HEADER
+doAll();
```
Comment (jgenso1690) [inline db/postgreSQL.js:49]: add an empty line at the end

Comment (AmiraVomir) [reply db/postgreSQL.js:49]: already has it

File: scripts/dataGen.js (hunk 1 above)
Comment (AmiraVomir) [reply scripts/dataGen.js:52]: already has it

**Pull request merged**